devenv/local/docker-compose/Dockerfile.emily

```diff
@@ -0,0 +1,72 @@
+FROM rust:bookworm as builder
+
+# Install dependencies.
+RUN apt-get update
+RUN apt-get install -y \
+    libclang-dev \
+    git pkg-config \
+    libssl-dev make \
+    protobuf-compiler \
+    npm \
+    default-jre
+RUN npm install -g pnpm@9
+RUN npm install -g @openapitools/openapi-generator-cli
+RUN rustup toolchain install stable
+RUN rustup component add rustfmt --toolchain stable
+# Setup the code.
+RUN mkdir /code
+WORKDIR /code
+RUN git clone https://github.com/stacks-network/sbtc.git
+WORKDIR /code/sbtc
+
+# TODO: DELETE THIS LINE BEFORE MERGING vvv
+RUN git checkout chore/run-signer-binary-against-devenv-emily-as-server
+# TODO: DELETE THIS LINE BEFORE MERGING ^^^
+
+RUN make install
+RUN make build
+
+# Generate CDK template
+# ------------------------------------------------------------------------------
+ARG AWS_STAGE=local
+ARG TABLES_ONLY=true
+RUN pnpm --filter emily-cdk run synth
+
+# NOTE: If you want to synthsize with the lambda and apigateway instance as well
+# you need to also compile the emily handler as a lambda and generate the openapi
+# spec before attempting to synthesize the cdk template.
+
+# Compile emily handler as a warp server.
+# ------------------------------------------------------------------------------
+RUN cargo build --release --bin emily-server
```

Comment:
We probably don't want release builds for `devenv`, though we probably do for testnet.

Reply:
Fair enough!